ukrainian/story/phone.md

```diff
@@ -0,0 +1,2 @@
+﻿Друг сказав , що вам слід скористатись ножем :)
+[Зробити отвір в кімнаті за допомогою ножа](story/knife.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.